api_server.py

```
@@ -549,4 +560,5 @@ def add_security_headers(response):
 """
 
 if __name__ == "__main__":
-    app.run(host="0.0.0.0", port=int(os.environ.get("PORT", 5000)))
+    # Host 0.0.0.0 is required for Docker container accessibility
+    app.run(host="0.0.0.0", port=int(os.environ.get("PORT", 5000)))  # nosec B104
```

Ruff (0.15.1)

[error] Possible binding to all interfaces

(S104)